build-system/tasks/runtime-test.js

```diff
@@ -203,7 +203,13 @@ gulp.task('test', 'Runs tests', argv.nobuild ? [] : ['build'], function(done) {
     util.log(util.colors.yellow(
         'Shutting down test responses server on localhost:31862'));
     server.emit('kill');
-    done(exitCode);
+    if (exitCode && typeof exitCode !== 'Error') {
```

Comment:
Yes, it works. Compare https://travis-ci.org/ampproject/amphtml/jobs/240439715 (with this change) against https://travis-ci.org/ampproject/amphtml/jobs/240195458 (without this change).

I wonder if it makes sense to set showStack to true. I figured I'd test it locally to see if it generates anything useful.